### PR TITLE
Feat: Implement Multi-Step Form Feature

### PR DESCRIPTION
**Pull request by ellaboevans**
This pull request contains the codebase for a multi-step form feature.

### CODE REVIEW

**Comment by fredrickyalley** [comment]
PR title

**Comment by ellaboevans** [comment]
> PR title

Sure. I have changed it to a more descriptive name.

**Comment by fredrickyalley** [comment]
When you don't fill first and second form but fill third form, you can go to the last form and complete the process which is wrong

**Comment by ellaboevans** [comment]
> When you don't fill first and second form but fill third form, you can go to the last form and complete the process which is wrong

This is true. I probably broke something. I am going to have it investigated and fixed.

**Comment by ellaboevans** [comment]
> > When you don't fill first and second form but fill third form, you can go to the last form and complete the process which is wrong
> 
> This is true. I probably broke something. I am going to have it investigated and fixed.

I have had this fixed @fredrickyalley

**Comment by fredrickyalley** [comment]
position the form input validation messages below the fields, it's more common that way

**Comment by ellaboevans** [comment]
> position the form input validation messages below the fields, it's more common that way

so this one was actually presented that way from the designs, hence the reason why I did but sure I will do exactly that.

**Comment by ellaboevans** [comment]
> Remove the hover on next button when disabled

Sure I will do exactly that as well.

**Comment by fredrickyalley** [comment]
your PR title isn't consistent with the previous PR structure

**Comment by ellaboevans** [comment]
> your PR title isn't consistent with the previous PR structure

I have fixed this.

File: week_5/multi-step-form/src/app/components/forms/forms.component.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  }
+
+  gotoNextStep() {
+    this.isForward = true;
+    if (this.currentStep === 1) {
+      const name = this.formFields.get('name');
+      const email = this.formFields.get('email');
+      const phone = this.formFields.get('phone');
+
+      if (name?.invalid || email?.invalid || phone?.invalid) {
```
Comment (fredrickyalley) [inline week_5/multi-step-form/src/app/components/forms/forms.component.ts:77]: create a variable for this OR conditions and use that here

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**